packages/web-components/src/customized-built-ins/image/image.stories.ts

```diff
@@ -0,0 +1,212 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../../helpers.stories.js';
+import { ImageFit, ImageShape } from '../../image/image.options.js';
+import { FUIImage } from './image.js';
+
+FUIImage;
```

Comment:
This registers the web component, right? Could you please elaborate on its relationship to `FluentDesignSystem.registry`? I would expect the component to be registered there.

Reply:
This actually just ensures that it doesn't get tree shaken out. The current implementation is using decorators, but we could probably use the `compose` and `define()` approach as well.